python/hail/api1/context.py

```diff
@@ -843,16 +843,30 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, min_p
     @handle_py4j
     @record_method
     @typecheck_method(path=oneof(strlike, listof(strlike)),
+                      key_expr=strlike,
```

Comment:
Minor point: can this be a `listof(strlike)` too?

Reply:
Withdrawn after conversation with @catoverdrive (sounds like no plans for composite keys on `MatrixTable` but instead I can do `key_expr='Locus(CHROM, POS)'` which works for me)

Reply:
@konradjk doing reviews now?  Awesome!

 > sounds like no plans for composite keys on MatrixTable

Yes, the plan is for composite keys on MatrixTable but there's some annoying infrastructure work to do first so key_expr was the short-term workaround.

Reply:
Agree with adding list(strlike)

Reply:
What would you want adding listof(strlike) do?

Reply:
Oh, wait, I take it back.  `key_expr` is a single expression, there's nothing to do.  @konradjk

Reply:
I guess my point is if you have a composite key, then you could specify it using `key_expr='CHROM, POS'` or `key_expr=['CHROM', 'POS']`, right?

Reply:
Yes, but we don't have a composite key yet.

Reply:
Yep, that's fine. Hence withdrawn 😄